.cirrus.yml

```diff
@@ -642,22 +706,18 @@ arm64_container_image_docker_builder:
   env:
     CIRRUS_ARCH: arm64
   << : *DOCKER_BUILD_TEMPLATE
-  << : *SKIP_TASK_ON_PR
+  << : *BUILDS_ONLY_IF_RELEASE_TAG_NIGHTLY
```

Comment:
Certainly for a follow-up: "Nightly if any changes since last time" would be really nice so we don't run tasks if there were no changes at all.